Website/Portals/_default/HotcakesViews/SocialSpokes/Views/Checkout/PaymentError.cshtml

```diff
@@ -29,7 +29,7 @@
             </div>
             <ul class="dnnActions">
                 <li>
-                    <input type="submit" id="btnsubmit" class="dnnPrimaryAction" value="@Localization.GetString("PlaceOrder")" tabindex="3001" />
+                    <input type="submit" id="hcTakeOrder" class="dnnPrimaryAction" value="@Localization.GetString("PlaceOrder")" tabindex="3001" />
```

Comment:
@VAlmea Is this a new update, or are you restoring it back to the hcTakeOrder class? What was the original value?

Reply:
I changed the btnsubmit Id to hcTakeOrder in the PaymentError view in order to reuse all the logic from the checkout view. I reviewed all the js files and the btnsubmit Id for the button is not used.